From: James Smart <james.smart@broadcom.com>
Date: Thu, 19 Nov 2020 12:33:28 -0800
Subject: scsi: lpfc: Fix missing prototype warning for
 lpfc_fdmi_vendor_attr_mi()
Patch-mainline: v5.11-rc1
Git-commit: 809032ddf9c6c779a9eb0f0c55c21a1e8913557f
References: bsc#1175480 bsc#1176396 bsc#1176942 bsc#1177500

Function needs to be declared as static.

Link: https://lore.kernel.org/r/20201119203328.121772-1-james.smart@broadcom.com
Fixes: 8aaa7bcf07a2 ("scsi: lpfc: Add FDMI Vendor MIB support")
Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: James Smart <james.smart@broadcom.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Acked-by: Daniel Wagner <dwagner@suse.de>
---
 drivers/scsi/lpfc/lpfc_ct.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/scsi/lpfc/lpfc_ct.c
+++ b/drivers/scsi/lpfc/lpfc_ct.c
@@ -3295,7 +3295,7 @@ lpfc_fdmi_smart_attr_security(struct lpf
 	return size;
 }
 
-int
+static int
 lpfc_fdmi_vendor_attr_mi(struct lpfc_vport *vport,
 			  struct lpfc_fdmi_attr_def *ad)
 {
